lib/server-starter.js

```diff
@@ -130,6 +150,15 @@ class Server extends EventEmitter {
     const port = this.port;
     return `http://${address}:${port}`;
   }
+
+  /**
+   * Listen Address to configure service to be launched
+   * @returns {string}
+   */
+  listenAddress () {
+    if (this._useFdPass) return 'http://*?fd=3';
+    return this.url();
+  }
```

Comment:
You're still missing the point. This module is not meant to be Mojolicious specific at all. This could be a generic feature that any other web framework could use just the same.